pylab/website/views.py

```diff
@@ -1,4 +1,5 @@
 from django.contrib import messages
+
```

Comment:
PEP-8 tells to group imports by category, all so all imports from Django should be grouped into one block, without extra spaces.

See: https://www.python.org/dev/peps/pep-0008/#imports